sdk/tables/Azure.Data.Tables/src/DynamicTableEntity.cs

```diff
@@ -0,0 +1,221 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using Azure.Core;
+
+namespace Azure.Data.Tables
+{
+
+    public partial class DynamicTableEntity
```

Comment:
It would be great to add some unit test coverage of this class as well.

Reply:
Nit - add a doc comment explaining what this is used for so it'll generate something nice in our reference documentation.  Possible simple example:

```C#
/// <summary>
/// Represents an untyped table entity.
/// </summary>
```